src/Illuminate/Container/Container.php

```diff
@@ -285,11 +286,19 @@ public function bindMethod($method, $callback)
      *
      * @param  array|string $method
      * @return string
+     *
+     * @throws \InvalidArgumentException
      */
     protected function parseBindMethod($method)
     {
         if (is_array($method)) {
-            return $method[0].'@'.$method[1];
+            if (count($method) > 1) {
+                return $method[0].'@'.$method[1];
+            }
+
+            throw new InvalidArgumentException(
+                sprintf('method should be array with 2 parameters, %s given', count($method))
```

Comment:
Use double quotes with `[$variable]` for variables.

Reply:
https://github.com/laravel/framework/blob/ba0fa733243fd9b5abac67e40c9a1f7ba2ca0391/src/Illuminate/Database/Query/JsonExpression.php#L43, https://github.com/laravel/framework/blob/ba0fa733243fd9b5abac67e40c9a1f7ba2ca0391/src/Illuminate/View/Engines/EngineResolver.php#L57, https://github.com/laravel/framework/blob/ba0fa733243fd9b5abac67e40c9a1f7ba2ca0391/src/Illuminate/Support/Manager.php#L97